docs/quickstart/index.rst

```diff
@@ -2,23 +2,34 @@
 Quickstart
 ==========
 
-Step 0: install package
+.. note::
+   These instructions describe a fully local workflow. To run on remote
+   resources, users may either `configure their resources themselves <https://qiskit-extensions.github.io/quantum-serverless/deployment/cloud.html>`_ or ask
+   their provider or dev ops team.
+
+Step 0: Close any running jupyter notebooks.
+
+Step 1: Install the quantum serverless package.
 
 .. code-block::
-   :caption: pip install
+   :caption: Install quantum_serverless via pip.
 
       pip install quantum_serverless==0.1.0
 
 
-Step 1: run infrastructure
+Step 2: Run infrastructure.
 
 .. code-block::
-   :caption: run docker compose from a root of the project
-
+   :caption: Run docker compose from the root of the quantum serverless project.
+      cd quantum-serverless/
       VERSION=0.1.0 docker-compose --profile full up
```

Comment:
@psschwei does this need to be bumped to 0.1.1 now that we have the latest from main in this branch?

Reply:
yes, we should bump _or_ merge #604 and then we can drop the version flag completely